<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>771</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="sendPage">
        <attribute name="title">
         <string>发送文件</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>711</width>
           <height>481</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>未刷新或局域网无其他设备接收IP默认本机 文件地址默认桌面</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>设备:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="RefreshIPButton">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>刷新设备</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="selectfile">
              <property name="text">
               <string>选择文件:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FileNameLineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="BrowserButton">
              <property name="text">
               <string>选择文件或拖入</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="receiveIP">
              <property name="text">
               <string>接收IP:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ReceiveIPLineEdit">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="port">
              <property name="text">
               <string>端口:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PortLineEdit">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="inputMethodHints">
               <set>Qt::InputMethodHint::ImhFormattedNumbersOnly</set>
              </property>
              <property name="inputMask">
               <string>99999</string>
              </property>
              <property name="text">
               <string>55555</string>
              </property>
              <property name="maxLength">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>200</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Sendbutton">
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>接收文件路径:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="savepathLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <property name="rightMargin">
             <number>480</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>本机接收监听端口:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="receiveportLineEdit">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="inputMethodHints">
               <set>Qt::InputMethodHint::ImhNone</set>
              </property>
              <property name="inputMask">
               <string>99999</string>
              </property>
              <property name="text">
               <string>55555</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="rightMargin">
             <number>520</number>
            </property>
            <item>
             <widget class="QLabel" name="tranlabel">
              <property name="text">
               <string>传输状态:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="statuslabel">
              <property name="text">
               <string>未开始</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="receivePage">
        <attribute name="title">
         <string>接收配置</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
